### PR TITLE
Update to v3.32.0

### PR DESCRIPTION
**Pull request by anton-z-s**

### CODE REVIEW

**Comment by anton-z-s** [comment]
Okay, it won't build with this fixes https://gitlab.gnome.org/GNOME/gitg/merge_requests/71

**Comment by anton-z-s** [comment]
Probably have to wait until https://gitlab.gnome.org/GNOME/gitg/merge_requests/71 is merged

**Comment by albfan** [comment]
Thanks for all the patches, I think we need to enable strict validation to avoid those metadata errors.

I will try to fix ASAP

File: org.gnome.gitg.json [hunk 1]
```diff
@@ -100,8 +98,8 @@
         "sources": [
             {
                 "type": "archive",
-                "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.0.tar.xz",
```
Comment (bilelmoussaoui) [inline org.gnome.gitg.json]: libgee is now on GNOME Sdk 3.32, you can remove it from here.

File: org.gnome.gitg.json [hunk 2]
```diff
@@ -133,11 +131,25 @@
...
+                "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz",
+                "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd"
             }
         ]
       },
+      {
+        "name": "libdazzle",
+        "config-opts": ["--libdir=/app/lib", "--buildtype=debugoptimized"],
+        "buildsystem": "meson",
+        "builddir": true,
```
Comment (bilelmoussaoui) [inline org.gnome.gitg.json]:
```suggestion
```

File: org.gnome.gitg.json [hunk 3]
```diff
@@ -133,11 +131,25 @@
...
-                "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.5.tar.xz",
-                "sha256": "0246185fcc20c4734d01419a83f58f251a82e2a902fe60bb0335187fcf658181"
+                "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz",
+                "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd"
             }
         ]
       },
+      {
+        "name": "libdazzle",
+        "config-opts": ["--libdir=/app/lib", "--buildtype=debugoptimized"],
```
Comment (bilelmoussaoui) [inline org.gnome.gitg.json]:
```suggestion
        "config-opts": ["--buildtype=debugoptimized"],
```

**Comment by albfan** [comment]
```
 argv: b'flatpak run org.freedesktop.appstream-glib validate builddir/*/share/appdata/org.gnome.gitg.appdata.xml'
 using PTY: False
builddir/files/share/appdata/org.gnome.gitg.appdata.xml: FAILED:
• tag-missing           : <name> is not present
• tag-missing           : <summary> is not present
Validation of files failed
```

https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Summary can be like description (or just first paragraph) and name "gitg"

Please create the patch and create a MR on gitg too (afk)

**Comment by albfan** [comment]
Thanks for all the hard work!

**Pull request merged**